types/pseudo-localization/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for pseudo-localization 2.1
+// Project: https://github.com/tryggvigy/pseudo-localization#readme
+// Definitions by: Jonathan Hensley <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/pseudo-localization/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me